controllers/user.controller.js

```diff
@@ -0,0 +1,107 @@
+const User = require("../models/user.model");
+const { extend } = require("lodash");
+
+const getUsers = async (req, res) => {
+  try {
+    let users = await User.find({});
+    users = users.map((user) => {
+      user.password = undefined;
+      return user;
+    });
+    res.json({ success: true, users });
+  } catch (err) {
+    res.status(500).json({
+      success: false,
+      message: "Unable to get the list of users",
+      errMessage: err.message,
+    });
+  }
+};
+
+const findUser = async (req, res) => {
+  const { username, password } = req.body;
+  const usernameExsists = await User.exists({ username });
+  if (usernameExsists) {
+    let user = await User.findOne({ username, password });
+    if (user) {
+      res.json({ success: true, user: { _id: user._id, name: user.name } });
+    } else {
+      res.status(401).json({
+        success: false,
+        message: "Username and password does not match",
+      });
+    }
+  } else {
+    res.status(401).json({
+      success: false,
+      message: "Username does not exsist",
```

Comment:
just a small typo here in the message content. Great work handling all user scenarios!